### PR TITLE
TGP 895 cya bug fix heading

### PR DESCRIPTION
**Pull request by besscerule**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -130,9 +130,20 @@ countryOfOrigin.inputHeading = Pick a country
...
+checkYourAnswers.title = Check your answers
+checkYourAnswers.h1 = Check your answers
+checkYourAnswers.h2 = UKIMS, NIRMS and NIPHL
+checkYourAnswers.ukimsNumber = UKIMS number
+checkYourAnswers.nirmsNumber = NIRMS number
+checkYourAnswers.nirmsRegistered = NIRMS registered
+checkYourAnswers.niphlNumber = NIPHL number
+checkYourAnswers.niphlRegistered = NIPHL registered
+
 checkYourAnswers.h2 = UKIMS, NIRMS and NIPHL
```
Comment (vinoocs) [inline conf/messages.en:143]: Duplicates may be

Comment (vinoocs) [inline conf/messages.en]: Duplicates may be, line 143,144 & 145

File: test/controllers/helpers/CheckYourAnswersHelperSpec.scala [hunk 2]
```diff
@@ -32,6 +32,11 @@ class CheckYourAnswersHelperSpec extends SpecBase with SummaryListFluency {
   when(fakeMessages("site.change")) thenReturn "Change"
```
Comment (ChrisCookOC) [inline test/controllers/helpers/CheckYourAnswersHelperSpec.scala:32]: Seems unnecessary to mock all the messages here? We can just use the stubbed messages in the SpecBase and check the result match the values like "site.change"

**Review by mycp98 — APPROVED** [review]

File: test/controllers/helpers/CheckYourAnswersHelperSpec.scala (hunk 2 above)
Comment (besscerule) [reply test/controllers/helpers/CheckYourAnswersHelperSpec.scala:32]: i agree! -changed! :)

Comment (besscerule) [reply test/controllers/helpers/CheckYourAnswersHelperSpec.scala:32]: actually - having checked your way makes the tests fail - if you can fix it better in your ticket feel free - but I'll pop this one in for now :)

**Pull request merged**